torch/_prims/context.py

```diff
@@ -229,6 +229,15 @@ def _is_var_mean(self, func):
             )
             and "aten.var_mean" in str(func)
         )
+    def _is_rand_like(self, func):
+        result = "torch.rand_like" == torch.overrides.resolve_name(func) or (
+            (
+                isinstance(func, torch._ops.OpOverload)
+                or isinstance(func, torch._ops.OpOverloadPacket)
+            )
+            and "aten.rand_like" in str(func)
```

Comment:
I should change it in my view PR. Let's state explicitly things we catch instead of substring matching
```suggestion
            func == torch.ops.aten.rand_like or func == torch.ops.aten.rand_like.default
```